ApplensBackend/Controllers/CaseCleansingController.cs

```diff
@@ -0,0 +1,150 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Authorization;
+using System.Data.SqlClient;
+using Dapper;
+using Dapper.Contrib.Extensions;
+using Kusto.Data.Net.Client;
+using System.Dynamic;
+using Kusto.Data;
+
+namespace AppLensV3.Controllers
+{
+    [Produces("application/json")]
+    [Route("api/casecleansing")]
+    [Authorize]
+    public class CaseCleansingController : Controller
+    {
+        public class CaseSimple
+        {
+            public string IncidentId { get; set; }
+            public DateTime Time { get; set; }
+            public string Status { get; set; }
+            public string AssignedTo { get; set; }
+            public DateTime ClosedTime { get; set; }
+            public int ID { get; set; }
+            public int RecommendationCount { get; set; }
+            public string Title { get; set; }
+        }
+
+        public class CaseExtra
+        {
+            public Recommendation[] Recommendations { get; set; }
+            public dynamic KustoData { get; set; }
+        }
+
+        public class Recommendation
+        {
+            public string RuleName { get; set; }
+            public string OldClosedAgainst { get; set; }
+            public string RecommendedClosedAgainst { get; set; }
+            public DateTime RecommendationDate { get; set; }
+        }
+
+        private ICaseCleansingClientService _caseCleansingService;
+
+        public CaseCleansingController(ICaseCleansingClientService caseCleansingService)
+        {
+            _caseCleansingService = caseCleansingService;
+        }
+
+        [HttpGet("GetAllCases")]
```

Comment:
I think all of our routes are lower case, just for consistency maybe we should make these lowercase also. Or at least camelCased